### DISCUSSION TITLE
port 8444 and load

### DISCUSSION THREAD

**Discussion by adriansilviu10** [Chia-Network/chia-blockchain#5741] [Support]
1. I can't open port 8444 ... I tried everything I could and it still doesn't work .... with version 1.1.3 I had no problems
2. 
is there any solution to get past this stage ??? I really don't know what to do to get out of here ....
![image_2021-05-21_212912](https://user-images.githubusercontent.com/78887197/119182441-96e0f100-ba7b-11eb-8650-537e45fc68cd.png)

**Comment by BrandtH22** [comment]
@adriansilviu10 , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
